scripts/deploy/riff-raff.yaml

```diff
@@ -7,9 +7,14 @@ deployments:
       templatePath: cloudformation.yml
       cloudFormationStackByTags: false
       cloudFormationStackName: rendering
+      templateStageParameters:
+        CODE:
+          InstanceType: t4g.micro
+        PROD:
+          InstanceType: t4g.small
       amiParametersToTags:
         AMI:
-          Recipe: dotcom-rendering-node14-test
+          Recipe: dotcom-rendering-ARM
```

Comment:
this saves us a job of switching back to the non-"testing" AMI 😆

Reply:
yep exactly